kuma/wiki/migrations/0001_initial.py

```diff
@@ -285,7 +284,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='document',
             name='team',
-            field=models.ForeignKey(blank=True, to='teamwork.Team', null=True),
+            field=models.ForeignKey(blank=True, to='wiki.DocumentTag', null=True),
```

Comment:
Looked like [this comment](https://github.com/mozilla/kuma/pull/3305#issuecomment-118953720) still needs to be addressed?

I like using the *not ready* label to help me ignore certain pull requests when doing a PR run, but if it's not helpful to the back-end team I can stop adding it when I triage our pull request backlog.